usr/lib/cca_stdll/ccatok.conf

```diff
@@ -0,0 +1,19 @@
+version cca-0
+
+# Optionally specify the expected master key verification patters for the 
+# SYM, AES, and APKA master key. The CCA token does not use the ASYM master key.
+#
+# You can use the TKE or panel.exe to query the curent master key verification
+# pattern:
```

Comment:
and of course the sysfs api also provides this info.
I still don't have this als part of lszcrypt but this comes soon.

Reply:
right, will add this info.